static/api/openapi.latest.json

```diff
@@ -2076,7 +2077,7 @@
           }
         },
         "security": [],
-        "summary": "Check if the server is running and ready to respond to requests."
+        "summary": "Test server status"
```

Comment:
Switched and shortened this because the summary is meant to be a short form field, description is long form.